.github/workflows/vercel-deploy-erd.yml

```diff
@@ -31,8 +31,9 @@ jobs:
         with:
           filters: |
             src:
-              - './.github/workflows/vercel-deploy-erd-sample.yml'
+              - './.github/workflows/vercel-deploy-erd.yml'
               - './frontend/apps/erd-sample/**'
+              - './frontend/apps/erd-web/**'
               - './frontend/packages/cli/**'
               - './frontend/packages/db-structure/**'
               - './frontend/packages/erd-core/**'
```

Comment:
Enabled `apps/erd-web` to deploy to the preview environment even when a PR contains changes only under `packages/`.